JPF/bad_buildings.sql

```diff
@@ -139,48 +175,64 @@ drop table if exists hs_permit_counts;
 ################
 create temporary table hs_permit_counts
 select 
-	bin_num,
-	CONCAT(trim(`house_num`)," ",trim(`street_name`)," ",trim(`borough`)) hs_dobp_address, 
+	bin_num bin,
+#	CONCAT(trim(`house_num`)," ",trim(`street_name`)," ",trim(`borough`)) address, 
+#   concat(trim(bl.boroid),trim(LPAD(dp.block, 5, '0')),trim(LPAD(dp.lot, 4, '0'))) bbl,
 	count(*) hs_permit_cnt
 from 
-	dob_permits 
+	dob_permits dp
+left join boro_lookup bl on bl.boro = dp.borough
 where 
 	work_type in ("OT","BL", "DM") 
 group by 
-	1,2;
-alter table hs_permit_counts add index(hs_dobp_address(255));
-alter table hs_permit_counts add index(bin_num);
+	1;
+#alter table hs_permit_counts add index(address(255));
+alter table hs_permit_counts add index(bin);
+#alter table hs_permit_counts add index(bbl);
 
 ################	
 drop table if exists permit_counts;
 ################
 create temporary table permit_counts
 select 
```

Comment:
What's the difference b/w permit_counts and hs_permit_counts? Just the work_type filtering? This really needs comments.